### PR TITLE
feat: handle legacy login form errors

### PR DESCRIPTION
**Pull request by NeOMakinG**
## Description
This PR is a follow-up PR of the login screen of legacy wallet, adding the error handling of the login page, waiting for the API implementation

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)
partially #1497

## Risk
- Not that risky, behind a feature flag

## Testing
- Try to login and see the error (or watch the screen)

## Screenshots (if applicable)
image

### CODE REVIEW

File: src/assets/translations/main.json [hunk 1]
```diff
@@ -757,6 +761,17 @@
...
+      "legacy": {
+        "haveMobileWallet": "Have a ShapeShift Mobile Wallet?",
+        "loginWithAccount": "Log In with ShapeShift Account",
+        "loginInformations": "Enter your email address and password you use to login to your ShapeShift Mobile App.",
+        "emailAddress": "Email Address",
+        "password": "Password",
+        "emailRequired": "Email is required",
+        "invalidEmail": "Email is invalid",
+        "invalidCaptcha": "Captcha is invalid",
+        "invalidLogin": "Invalid login"
```
Comment (theoboldfrazier) [inline src/assets/translations/main.json:774]:
```suggestion
        "invalidLogin": "Invalid login"
```
```suggestion
        "invalidLogin": "Invalid login",
        "twoFactorRequired": "2FA is required"
```

Comment (theoboldfrazier) [inline src/assets/translations/main.json:774]:
```suggestion
        "invalidLogin": "Invalid login"
```
```suggestion
        "invalidLogin": "Invalid login",
        "twoFactorRequired": "2FA is required"
```

Comment (theoboldfrazier) [reply src/assets/translations/main.json:774]: Not sure why there are multiple suggestions/comments. Just wanted to suggest we'll need a 2FA error message.

Comment (theoboldfrazier) [reply src/assets/translations/main.json:774]: Not sure why there are multiple suggestions/comments. Just wanted to suggest we'll need a 2FA error message.

**Review by theoboldfrazier — APPROVED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**